### PR TITLE
Add rbd, subvolume and listing of pool names command in ceph must-gather

### PR DESCRIPTION
**Pull request by crombus**
https://bugzilla.redhat.com/show_bug.cgi?id=1806953

### CODE REVIEW

File: must-gather/collection-scripts/gather_ceph_resources [hunk 1]
```diff
@@ -100,6 +103,7 @@ for ns in $namespaces; do
                 { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "${ceph_commands[$i]} --connect-timeout=15" >> "${COMMAND_OUTPUT_FILE}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
                 { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "${ceph_commands[$i]} --connect-timeout=15 --format json-pretty" >> "${JSON_COMMAND_OUTPUT_FILE}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
             done
+            for i in `timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph osd lspools --connect-timeout=15"|awk '{print $2}'`; do { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd ls -p $i" >> ${COMMAND_OUTPUT_DIR}/pools_rbd_$i; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1; done
```
Comment (jarrpa) [inline must-gather/collection-scripts/gather_ceph_resources:106]: Why is this all in one line? Expand it across multiple lined if possible.

Comment (jarrpa) [inline must-gather/collection-scripts/gather_ceph_resources:106]: Why are you running the `ceph osd lspools` command again? Can't you just read the command output file?

File: must-gather/collection-scripts/gather_ceph_resources [hunk 2]
```diff
@@ -49,6 +49,9 @@ ceph_commands+=("ceph mds stat")
 ceph_commands+=("ceph versions")
 ceph_commands+=("ceph fs dump")
 ceph_commands+=("ceph auth list")
+ceph_commands+=("ceph osd lspools")
```
Comment (travisn) [inline must-gather/collection-scripts/gather_ceph_resources:52]: This should give more useful details: `ceph osd pool ls detail`

File: must-gather/collection-scripts/gather_ceph_resources (hunk 1 above)
Comment (crombus) [reply must-gather/collection-scripts/gather_ceph_resources]: I was unable to directly  consume the command output file. This way it was less code of line and variables. 
e.g. while read line; do rbd ls -p $line |awk '{print $2}' ; done < ceph_osd_pools
 the awk command doesn't work like this. if we want to read the output file I can find the alternative for it.

**Pull request merged**